InvenTree/common/models.py

```diff
@@ -1395,6 +1403,184 @@ def is_valid_choice(cls, user_color_theme):
         return False
 
 
+class VerificationMethod:
+    NONE = 0
+    TOKEN = 1
+    HMAC = 2
+
+
+class WebhookEndpoint(models.Model):
```

Comment:
Should the "WebhookEndpoint" have a foreignkey relation to "PluginConfig"? This way, when the plugin is uninstalled, so are the corresponding endpoints. Also, it would enable plugins to keep track of their own endpoints?